velox/functions/sparksql/DateTimeFunctions.h

```diff
@@ -172,10 +172,13 @@ struct UnixTimestampParseFunction {
       const std::vector<TypePtr>& /*inputTypes*/,
       const core::QueryConfig& config,
       const arg_type<Varchar>* /*input*/) {
-    format_ = detail::getDateTimeFormatter(
+    auto formatter = detail::getDateTimeFormatter(
         kDefaultFormat_,
         config.sparkLegacyDateFormatter() ? DateTimeFormatterType::STRICT_SIMPLE
                                           : DateTimeFormatterType::JODA);
+    // Default format should always be valid.
+    VELOX_CHECK(!formatter.hasError());
```

Comment:
Perhaps make 'Default format should always be valid.' the error message of VELOX_CHECK.

Reply:
Thanks, done.